packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -557,13 +577,16 @@ export default decorate([
       missingRemotes: state =>
         (state.backupMode || state.deltaMode) && isEmpty(state.remotes),
       missingSrs: state => (state.drMode || state.crMode) && isEmpty(state.srs),
-      missingSchedules: state => isEmpty(state.schedules),
-      missingExportRetention: state =>
-        state.exportMode && !state.exportRetentionExists,
-      missingCopyRetention: state =>
-        state.copyMode && !state.copyRetentionExists,
-      missingSnapshotRetention: state =>
-        state.snapshotMode && !state.snapshotRetentionExists,
+      missingSchedules: (state, { job }) =>
+        job !== undefined && isEmpty(state.schedules),
```

Comment:
Shouldn't we remove this?

Reply:
This functionality is only for backup creation.

Reply:
What do you mean? Do you mean it should still show an error when you edit a backup job and remove all its schedules?

Reply:
Yes, IMHO, it will not have sense to delete all existent schedules and replace them by a default schedule. It can certainly be a user mistake.

Reply:
Yes, I agree.

Reply:
But when you edit a schedule, the user can't remove all the schedules, right? When there's only one left, the delete button is disabled.

Reply:
By the way, maybe we should let the user remove the last schedule and keep the error message. Sometimes you just want to remove all the schedules and create a new one with the default values.

Reply:
LGTM :+1: